DEVELOPMENT.md

```diff
@@ -0,0 +1,84 @@
+# Development
+
+This doc explains how to set up a development environment, so you can get started
+contributing to `oam-kubernetes-runtime` or build a PoC (Proof of Concept). 
+
+## Prerequisites
+
+1. Golang version 1.12+
+2. Kubernetes version v1.15+ with kubectl configured
+
+## Build
+
+The functionality of this library can be demonstrated with the following steps:
+
+* Fork and clone this project
+```shell script
+git clone https://github.com/xxxxxx/oam-kubernetes-runtime.git
+```
+
+* Build the library 
+
+```shell
+make submodules 
+
+make
+```
+
+## Make your change
+* Generate and install CRDs to your Kubernetes cluster
+
+```shell
+make generate
+
+kubectl apply -f crds/
+```
+
+* Make your code changes
+
+Make changes and run the following to test it.
+
```

Comment:
* `make reviewable` will generate and lint the code.
* `make test` will run tests.

Make sure these two steps are executed after every change.